presto-docs/src/main/sphinx/release/release-0.101.rst

```diff
@@ -12,6 +12,7 @@ General Changes
 * Add :func:`array_position` function.
 * Add :func:`regexp_split` function.
 * Fix analysis of ``UNION`` queries for tables with hidden columns.
+* Fix precedence of ``JOIN`` when more than one ``JOIN`` exists in a query.
```

Comment:
Give a little more detail on this. Does it effect all queries that have multiple JOINs?